### PR TITLE
[HAMMER] [Changelog] Update for Hammer Beta-1 release

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

### CODE REVIEW

**Comment by JPrause** [comment]
@miq-bot add_label documentation

**Comment by JPrause** [comment]
@miq-bot assign simaishi

File: CHANGELOG.md [hunk 1]
```diff
@@ -286,71 +259,4 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
-- Adding custom_actions support for /api/cloud_networks [(#200)](https://github.com/ManageIQ/manageiq-api/pull/200)
-- Add support for /api/switches [(#191)](https://github.com/ManageIQ/manageiq-api/pull/191)
-- Add support for /api/container_volumes [(#190)](https://github.com/ManageIQ/manageiq-api/pull/190)
-- Add support for /api/container_templates [(#188)](https://github.com/ManageIQ/manageiq-api/pull/188)
-- Adding support for /api/container_images [(#185)](https://github.com/ManageIQ/manageiq-api/pull/185)
-- Adding support for /api/container_groups [(#184)](https://github.com/ManageIQ/manageiq-api/pull/184)
-- Adding support for /api/cloud_object_store_containers [(#183)](https://github.com/ManageIQ/manageiq-api/pull/183)
-- Add custom action support for models already exposed in api that need it [(#163)](https://github.com/ManageIQ/manageiq-api/pull/163)
-
-## Unreleased as of Sprint 72 ending 2017-10-30
```
Comment (simaishi) [inline CHANGELOG.md]: This `Sprint 72 ending 2017-10-30` section should be a part of Gaprindashvili release, as we branched Gaprindashvili on Oct 31.

Comment (JPrause) [reply CHANGELOG.md]: thanks Satoe. Corrected now.

File: CHANGELOG.md [hunk 2]
```diff
@@ -33,30 +111,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Set user when queueing VM actions [(#326)](https://github.com/ManageIQ/manageiq-api/pull/326)
 - Symbolize schedule data [(#436)](https://github.com/ManageIQ/manageiq-api/pull/436)
 
-## Unreleased as of Sprint 93 ending 2018-08-27
```
Comment (bdunne) [inline CHANGELOG.md]: Are these changes not included in this branch?

File: CHANGELOG.md [hunk 3]
```diff
@@ -71,37 +125,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
-- Ensure refresh => true option gets passed for service_template refresh [(#386)](https://github.com/ManageIQ/manageiq-api/pull/386)
-- use request_admin_user? [(#385)](https://github.com/ManageIQ/manageiq-api/pull/385)
-
-## Unreleased as of Sprint 86 ending 2018-05-21
-
-### Added
-- Adds request_retire action for Vms and Services [(#380)](https://github.com/ManageIQ/manageiq-api/pull/380)
-- Creating PhysicalChassis controller and adding endpoint configuration [(#362)](https://github.com/ManageIQ/manageiq-api/pull/362)
-
 ## Gaprindashvili-3 released 2018-05-15
```
Comment (bdunne) [inline CHANGELOG.md]: How did this get here?

**Comment by simaishi** [comment]
@JPrause This PR also has issues with some Gaprindashvili backported PRs being in Hammer section... it's probably true for other changelog PRs too, so I'm going to merge this for now and you can probably revisit them all later and fix.

**Pull request merged**